xml/CRSChannelCode.xml

```diff
@@ -9,2494 +9,3284 @@
             </name>
+            <category>21000</category>
         </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
-            <code>21029</code>
-            <name>
-                <narrative>Doctors Without Borders</narrative>
-                <narrative xml:lang="fr">Médecins Sans Frontières</narrative>
-            </name>
-        </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
-            <code>47035</code>
+        <codelist-item status="active" activation-date="2011-01-01">
+            <code>21006</code>
             <name>
-                <narrative>Environmental Development Action in the Third World</narrative>
-                <narrative xml:lang="fr">Environnement et développement du Tiers-monde</narrative>
+                <narrative>Development Gateway Foundation</narrative>
+                <narrative xml:lang="fr">Development Gateway Foundation</narrative>
             </name>
+            <category>22000</category>
         </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
+        <codelist-item status="withdrawn" activation-date="2010-01-01" withdrawal-date="2010-12-31">
             <code>21007</code>
             <name>
                 <narrative>Environmental Liaison Centre International</narrative>
                 <narrative xml:lang="fr">Centre international de liaison pour l’environnement</narrative>
             </name>
+            <category>21000</category>
         </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
-            <code>21503</code>
+        <codelist-item status="active" activation-date="2011-01-01">
+            <code>21008</code>
             <name>
-                <narrative>Family Health International 360</narrative>
+                <narrative>Eurostep</narrative>
+                <narrative xml:lang="fr">Eurostep</narrative>
+            </name>
+            <category>22000</category>
+        </codelist-item>
+        <codelist-item status="active" activation-date="2011-01-01">
+            <code>21009</code>
+            <name>
+                <narrative>Forum for Agricultural Research in Africa</narrative>
+                <narrative xml:lang="fr">Forum pour la recherche agricole en Afrique</narrative>
+            </name>
+            <category>51000</category>
+        </codelist-item>
+        <codelist-item status="active" activation-date="2011-01-01">
+            <code>21010</code>
+            <name>
+                <narrative>Forum for African Women Educationalists</narrative>
+                <narrative xml:lang="fr">Forum des éducatrices africaines</narrative>
             </name>
+            <category>23000</category>
         </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
+        <codelist-item status="withdrawn" activation-date="2010-01-01" withdrawal-date="2010-12-31">
```

Comment:
I don’t think this codelist item has been withdrawn. There are two withdrawn entries for this item, but also one active one.

Reply:
Looking at the source XML, code 21011 does indeed appear three times. Active from January 1, 2003, withdrawn on December 31, 2003; reactivated January 1, 2004 (without an in-line withdrawal); and then reactivated again on January 1, 2010, withdrawn December 31, 2010.

Maybe it's worth asking the OECD about the intended meaning, @PetyaKangalova ? I think it would be fair to concatenate codelist items when codes and narratives match verbatim (which would ultimately make this item withdrawn, with an activate date of 2003 and withdrawal date at the end of 2010?), but I don't want to make that determination for them. I'm also assuming that we don't want codelist items to have noncontiguous active time periods.

Reply:
Thanks @akmiller01 ! Will check on this one.

Reply:
> which would ultimately make this item withdrawn, with an activate date of 2003 and withdrawal date at the end of 2010?

I don’t think that’s necessarily the case. It completely depends on the rules you decide to apply.

It’s also worth noting that in this instance, this code is present in the XLS:
![Screenshot 2020-03-13 at 10 39 57](https://user-images.githubusercontent.com/464193/76613935-01f97400-6517-11ea-8b99-1dfeb332b0b8.png)

Anyway – thanks for checking with the DAC on this.

Reply:
Thanks @andylolz this code has been reactivated.